app/resources/views/pages/location_report/collapsed.blade.php

```diff
@@ -27,6 +36,10 @@
 		</div>
 		<div class="location-tags text-right">
 			<div class="location-tags">
+			<a class="location-tag" title="Update Information" href="#" data-toggle="modal" data-target="#suggestionModal">
```

Comment:
Put an if-statement around this button so it doesn't show unless the user is signed in.  It doesn't work for users who haven't signed in so not showing the button should prevent some confusion and frustration for them.

The home.blade.php does something similar for its Sign In and Sign Up buttons which are hidden when people _ARE_ signed in.